src/app/modules/content/knowledge-component/knowledge-component.component.html

```diff
@@ -0,0 +1,58 @@
+<mat-sidenav-container hasBackdrop="false">
+  <mat-sidenav fixedInViewport="true" fixedTopGap="65" mode="side" position="end" [opened]="sidenavOpened">
+    <button mat-raised-button mat-icon-button color="primary" (click)="sidenavOpened = false">
+      <mat-icon>keyboard_arrow_right</mat-icon>
+    </button>
+    <h2 class="mat-h2">
+      Notes
+    </h2>
+    <cc-notes></cc-notes>
+  </mat-sidenav>
+  <mat-sidenav-content>
+    <div id="knowledge-component" *ngIf="knowledgeComponent">
+      <br>
+      <button id="ae-button" mat-raised-button
+              [ngStyle]="{'background-color': instructionalEventChecked ? '#04234e' : '#031e43'}"
+              (click)="onAssessmentEventClicked()">AE
+      </button>
+      <button id="ie-button" mat-raised-button
+              [ngStyle]="{'background-color': instructionalEventChecked ? '#031e43' : '#04234e'}"
+              (click)="onInstructionalEventClicked()">IE
+      </button>
+      <h3>Goal: {{knowledgeComponent.name}}</h3>
+      <mat-divider></mat-divider>
+      <cc-learning-object-container
+        *ngFor="let learningObject of learningObjects"
+        [learningObject]="learningObject"></cc-learning-object-container>
+      <br>
+      <!--      <div id="navigation" fxLayoutGap="20px">-->
```

Comment:
Remove. On our next sync we should discuss the interaction pattern and design the UI.